Source/common/SNTCommon.h

```diff
@@ -45,15 +46,22 @@ typedef enum {
    x == ACTION_RESPOND_ALLOW_COMPILER)
 
 // Struct to manage vnode IDs
-typedef struct santa_vnode_id_t {
+typedef struct SantaVnode {
   uint64_t fsid;
   uint64_t fileid;
```

Comment:
WDYT about using this as a chance to chance the type of these to `st_dev` and `st_ino` so we don't need any casting?